frontend/src/__tests__/cypress/cypress/tests/mocked/modelCatalog/modelCatalog.cy.ts

```diff
@@ -32,25 +53,47 @@ const initIntercepts = ({ disableModelCatalogFeature = false }: HandlersProps) =
     }),
   );
 
+  // Mock the managed ConfigMap (Red Hat sources)
   cy.interceptK8s(
     {
       model: ConfigMapModel,
       ns: 'opendatahub',
       name: 'model-catalog-sources',
     },
-    mockModelCatalogConfigMap(),
+    mockManagedModelCatalogConfigMap(managedSources),
   );
+
+  if (hasUnmanagedSourcesConfigMap) {
+    cy.interceptK8s(
+      {
+        model: ConfigMapModel,
+        ns: 'opendatahub',
+        name: 'model-catalog-unmanaged-sources',
+      },
+      mockUnmanagedModelCatalogConfigMap(unmanagedSources),
+    );
+  } else {
+    cy.interceptK8s(
+      {
+        model: ConfigMapModel,
+        ns: 'opendatahub',
+        name: 'model-catalog-unmanaged-sources',
+      },
+      mockConfigMap404Response('model-catalog-unmanaged-sources'),
+    );
+  }
+
+  cy.interceptK8sList(ServiceModel, mockK8sResourceList(modelRegistries));
 };
 
 describe('Model Catalog core', () => {
   it('Model Catalog Disabled in the cluster and URLs should not exist', () => {
     initIntercepts({
       disableModelCatalogFeature: true,
+      hasUnmanagedSourcesConfigMap: false,
     });
 
-    modelCatalog.landingPage();
-
-    cy.findByRole('button', { name: 'Models' }).click();
+    cy.visit('/');
```

Comment:
I think you still need the `cy.findByRole('button', { name: 'Models' }).click();` here right? since it has to open the models menu in the nav to check if its children are there (even though it'll be not there either way, this test isn't testing exactly what you intend).

Also, why remove the `modelCatalog.landingPage()` call? It does the `visitWithLogin()` and has its `wait` command that I think are maybe necessary

Reply:
I'm so sorry @YuliaKrimerman 😅 with `landingPage` added back here you don't need `cy.visit('/')`, landingPage calls that via its `visitWithLogin('/')`